### DISCUSSION TITLE
xrdp and gnome?

### DISCUSSION THREAD

**Discussion by DhaosEsedess** [WhitewaterFoundry/Fedora-Remix-for-WSL#196] [General]
anyone have any luck getting this to work on fedora remix? If so, could I get a walkthrough? and, what about KVM on fedora remix?

**Comment by crramirez** [comment]
Hello,

Here is the walk through https://www.whitewaterfoundry.com/blog/2022/4/16/announcing-built-in-systemd-support-in-fedora-remix-for-wsl we are updating it to Fedora 38 but still works.

You'll need to add

```
[general]
guiApplications=false

[boot]
systemd=true
```
To the /etc/wsl.conf file and restart it with `wsl.exe --terminate fedoraremix`

Then follow the walk through

**Comment by crramirez** [comment]
About KVM, WSL2 comes with nested virtualization enabled by default. Unless you have AMD and Windows 10, in this case it won't work, because nested virtualization support for AMD was introduced in Windows 11
